bokehjs/test/common/defaults.coffee

```diff
@@ -62,9 +62,17 @@ check_matching_defaults = (name, python_defaults, coffee_defaults) ->
       continue
 
     # special case for date time tickers, class hierarchy and attributes are handled differently
-    if name = "DatetimeTicker" and k = "tickers"
+    if name == "DatetimeTicker" and k == "tickers"
       continue
 
+    # special case for Title derived text properties
+    if name == "Title" and (k == "text_align" or k == "text_baseline")
+      continue
+
+    # special case for selections that have a method added to them
+    if k == 'selected'
+      v['0d'] = _.omit(v['0d'], 'get_view')
+
```

Comment:
ping @almarklein if there is something better we can do, please let me know. Without this, the `get_view` key shows up in JS but not python, so there is a mismatch and error